Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -227,6 +227,78 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		1001E2B62CF643270023A03C /* CardVault */ = {
+			isa = PBXGroup;
+			children = (
+				1001E2BD2CF644E00023A03C /* CardVaultViewModel */,
+				3B43290F2A8FD7FD00C5441A /* CardVaultViews */,
+			);
+			path = CardVault;
+			sourceTree = "<group>";
+		};
+		1001E2B72CF6432F0023A03C /* PayPalVault */ = {
+			isa = PBXGroup;
+			children = (
+				1001E2BC2CF644BE0023A03C /* PayPalVaultViewModel */,
+				3B2501052B2679F000903EAB /* VaultViews */,
+				3BB60B512B1F9EE400A298CF /* PayPalVaultViews */,
+			);
+			path = PayPalVault;
+			sourceTree = "<group>";
+		};
+		1001E2B82CF643350023A03C /* CardPayments */ = {
+			isa = PBXGroup;
+			children = (
+				1001E2BB2CF6449B0023A03C /* CardPaymentViewModel */,
+				3BA56FEA2A9DCBB30081D14F /* CardPaymentViews */,
+			);
+			path = CardPayments;
+			sourceTree = "<group>";
+		};
+		1001E2B92CF6433C0023A03C /* PayPalWebPayments */ = {
+			isa = PBXGroup;
+			children = (
+				1001E2BA2CF644480023A03C /* PayPalWebViewModel */,
+				3BA56FFF2A9FF6630081D14F /* PayPalWebPaymentsView */,
+			);
+			path = PayPalWebPayments;
+			sourceTree = "<group>";
+		};
+		1001E2BA2CF644480023A03C /* PayPalWebViewModel */ = {
+			isa = PBXGroup;
+			children = (
+				3BA56FFB2A9FEFE90081D14F /* PayPalWebViewModel.swift */,
+			);
+			path = PayPalWebViewModel;
+			sourceTree = "<group>";
+		};
+		1001E2BB2CF6449B0023A03C /* CardPaymentViewModel */ = {
+			isa = PBXGroup;
+			children = (
+				3BA56FE62A9DC9D70081D14F /* CardPaymentViewModel.swift */,
+				3BA56FE82A9DCA520081D14F /* CardPaymentState.swift */,
+			);
+			path = CardPaymentViewModel;
+			sourceTree = "<group>";
+		};
+		1001E2BC2CF644BE0023A03C /* PayPalVaultViewModel */ = {
+			isa = PBXGroup;
+			children = (
+				3BB60B542B1FA00C00A298CF /* PayPalVaultViewModel.swift */,
+				3BA0A58A2B1E240300330681 /* VaultViewModel.swift */,
+				3B2027402A8A72050007907E /* VaultState.swift */,
```

Comment:
VaultViewModel is a parent class and VaultState is a published state in VaultViewModel, maybe they could be in their own folder?

Reply:
![Screenshot 2024-11-26 at 11 04 24 AM](https://github.com/user-attachments/assets/4d0c9dd1-b849-44e7-b0b1-4e9cbe7eaafa)
The PayPalVault -> PayPalVaultViewModel => VaultState, VaultViewModel
PayPalVault -> views under VaultViews, these are common views for merchant side set up for all vault flows.

Could go under just "Vault" folder

Reply:
oh I meant outside of the PayPalVault folder. There is PayPal vault, Card vault and other payment methods in the future.
And how about we order the views in sequence of expected events of the main view?

Reply:
so for example under CardPayments, there are container views like CardPaymentView (which is not a good name, originally it was going to be container view for all the subviews like in CardVault) which has create order, order result result, the SDK portion, approve and result and container view for order completion with capture/authorize and result view

Reply:
> so for example under CardPayments, there are container views like CardPaymentView (which is not a good name, originally it was going to be container view for all the subviews like in CardVault) which has create order, order result result, the SDK portion, approve and result and container view for order completion with capture/authorize and result view

maybe i can change the name to CardViewComponents or CardPaymentsScreens ?

Reply:
CardPaymentView => that actually is the main container view.
CreateOrderCardPaymentView
OrderCreateCardResultView
CardOrderApproveView
CardApprovalResultView
CardPaymentOrderCompletionView
CardOrderActionButton
CardOrderCompletionResultView

Reply:
I think CardPaymentView is ok, consistent with PayPalVaultView. I like CardPaymentsScreens. 
but we have PayPalWebButtonsView as main container view in PayPalWebPaymentsView.

Reply:
Not too particular, I think having the correct order would make it easier to read.

Reply:
Sure! I prefer CardPaymentsScreens over CardViewComponents because it's clearer about which module the container view is for.